pandas/core/interchange/from_dataframe.py

```diff
@@ -298,13 +298,14 @@ def string_column_to_ndarray(col: Column) -> tuple[np.ndarray, Any]:
 
     null_pos = None
     if null_kind in (ColumnNullType.USE_BITMASK, ColumnNullType.USE_BYTEMASK):
-        assert buffers["validity"], "Validity buffers cannot be empty for masks"
```

Comment:
this isn't true for pyarrow dtypes right? they use bitmasks, but their validity buffer can indeed be `None`, whereas pandas nullables always seem to be set?
```
In [7]: pd.Series([1, 2, 3], dtype='Int64').array._mask
Out[7]: array([False, False, False])

In [8]: pd.Series([1, 2, 3], dtype='Int64[pyarrow]').array._pa_array.chunks[0].buffers()[0] is None
Out[8]: True
```